Jenkinsfile

```diff
@@ -2,12 +2,14 @@
 
 @Library('pipeline-library') _
 
-def img
+def image
```

Comment:
Variable name `img` has been replaced with `image`. Sensible change.

This is used to store the `h` docker image we have always created.